src/angular-cesium/models/context-menu-options.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Interface of context menu options that can be passed when opening a context menu.
+ */
+
+export interface ContextMenuOptions {
+  data?: any;
+  closeOnLeftCLick?: boolean;
```

Comment:
instead of left click allow the user to pass close event he wants

Reply:
I dont think there is a need for a specific event. the user should open the context menu whenever he needs to - (same as material modal). i just added a default logic to make things a little bit easier for users.